README.md

```diff
@@ -1 +1,81 @@
+<p align="center">
+  <img src="client/src/media/nodecloudlogo.png">
+</p>
+
 # nodecloud-web
+
+nodecloud-web provides an interactive front-end for [nodecloud](https://github.com/cloudlibz/nodecloud). NodeCloud is a standard library to get a single API on the open cloud with multiple providers.
+Making open cloud easily accessible and managed.
+
+![screenshot](screenshots/screenrecord.gif)
+
+# clone or download
```

Comment:
Let's use header level 2 i.e ## for sub headings like this.

Reply:
@rajikaimal I have made the requested changes, please check.